kernel/init.c

```diff
@@ -186,11 +157,17 @@ static void _main(void *unused1, void *unused2, void *unused3)
 
 	_sys_device_do_config_level(_SYS_INIT_LEVEL_POST_KERNEL);
 	if (boot_delay > 0) {
-		printk("***** delaying boot " STRINGIFY(CONFIG_BOOT_DELAY)
-		       "ms (per build configuration) *****\n");
+		SYS_LOG_INF("delaying boot " STRINGIFY(CONFIG_BOOT_DELAY)
```

Comment:
nit: consider use %d instead of STRINGIFY to be consistent with the following code.